python/deltalake/table.py

```diff
@@ -1176,7 +1176,10 @@ def with_writer_properties(
         write_batch_size: Optional[int] = None,
         max_row_group_size: Optional[int] = None,
     ) -> "TableMerger":
-        """Pass writer properties to the Rust parquet writer, see options https://arrow.apache.org/rust/parquet/file/properties/struct.WriterProperties.html:
+        """
+        !!! warning "Deprecated"
+            Use `.merge(writer_properties = WriterProperties())` instead
+        Pass writer properties to the Rust parquet writer, see options https://arrow.apache.org/rust/parquet/file/properties/struct.WriterProperties.html:
```

Comment:
Can we use something other than "Rust Parquet writer" for the Python API docs?

I would rather link to our docs than a third party page?  Can we add a "Python version" of all these options to the documentation?

Reply:
There is a short description of each argument, I only kept the rust docs link in there in case people want to request more properties to be exposed.

Reply:
@MrPowers what shall we do? Would be good to merge this since the new release contains the WriterProperties but the docs not yet if we don't merge this : P

Reply:
@ion-elgreco - merge away!  We can always improve the docs later!